### ISSUE TITLE
That's when it comes to concatenating variables.

### ISSUE THREAD

**Issue by allacmc** [RalphBacon/224-Superior-Serial.print-statements#4]
The problem with this implementation.

That's when it comes to concatenating variables.

This command does not support example:

Serial.println("[INFO] - Conectando no WiFi " + Wifi_SSID + ".");

**Comment by RalphBacon** [comment]
True. But you could create a String variable (much as I hate them due to the fact they will fragment the heap memory) and then  print that.

String msg = "[INFO] - Conectando no WiFi " + Wifi_SSID + ".";
debugPrintln(msg);

Tudo bem!
